collect_app/src/main/java/org/odk/collect/android/activities/viewmodels/CurrentProjectViewModel.kt

```diff
@@ -0,0 +1,25 @@
+package org.odk.collect.android.activities.viewmodels
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import org.odk.collect.android.projects.CurrentProjectProvider
+import org.odk.collect.androidshared.data.Consumable
+import org.odk.collect.projects.Project
+
+class CurrentProjectViewModel(currentProjectProvider: CurrentProjectProvider) : ViewModel() {
+
+    private val _currentProject = MutableLiveData(Consumable(currentProjectProvider.getCurrentProject()!!))
```

Comment:
I think we should probably make `getCurrentProject` return `String` rather than `String?`. We really don't expect it to be null. I think a check and throw in `getCurrentProject` (or just a `!!` in there) would be nice.

Reply:
It returns `Project?` not `String?` but I agree this shouldn't return null. I just had problems fixing it now because of those tests but I'll do that as a part of the next pr.

Reply:
Ach sorry I'm getting distracted by the `?` and missing the important part there.